### ISSUE TITLE
Potential false positive in wai-aria check with `<select multiple>`

### ISSUE THREAD

**Issue by kmoschcau** [markuplint/markuplint#1084]
- Markuplint version: `3.11.0`
- Parser lang: HTML
- Node.js version: `19.6.0`
- OS: WSL2 on Windows 10

## Describe the bug

markuplint complains about a `<select multiple>` with `<option>`s inside:

`The "listbox" role expects the roles: "group > option", "option" (wai-aria)`

If I am understanding the spec correctly, `<option>`s should always have the `option` role implicitly. Yet somehow this check isn't picking up on it. If this is intentional, please give me a hint on how to fix it. I've tried `<optgroup>` and `<datalist>` in the `<select>` and neither worked.

## Code Example

```html
<select multiple aria-label="some label text">
  <option>foo</option>
  <option>bar</option>
</select>
```

## Steps To Reproduce

1. Run markuplint over the above code.

## The current behavior

It shows the above complaint.

## The expected behavior

It shouldn't complain.

**Comment by YusukeHirao** [comment]
@kmoschcau 

Thank you for the report.
I'll check it.
